build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
 		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
 	}
 	developmentOnly("org.springframework.boot:spring-boot-devtools")
+    implementation 'org.jetbrains:annotations:15.0'
```

Comment:
Not sure where this line came from. Do you know if it's wrong?

Reply:
Probably you added an annotation that belongs to that package, and given it's an IntelliJ annotation, might probably have added this to your build.gradle automatically.